questionary/prompts/common.py

```diff
@@ -1,14 +1,16 @@
 # -*- coding: utf-8 -*-
 import inspect
 from prompt_toolkit import PromptSession
-from prompt_toolkit.filters import IsDone, Always
+from prompt_toolkit.filters import Always, Condition, IsDone
+from prompt_toolkit.keys import Keys
 from prompt_toolkit.layout import (
     FormattedTextControl,
     Layout,
     HSplit,
     ConditionalContainer,
     Window,
 )
+from prompt_toolkit.layout.dimension import LayoutDimension as D
```

Comment:
I think it would be better not to import as `D` and instead use the full `LayoutDimension` identifier.